internal/provider/resource_access_flow.go

```diff
@@ -131,8 +128,27 @@ func (a accessFlowResource) Schema(_ context.Context, _ resource.SchemaRequest,
 			},
 			"grantees": schema.SetNestedAttribute{
 				MarkdownDescription: "Represents which identities should be granted access",
-				Required:            true,
+				Optional:            true,
 				NestedObject:        identitySchema,
+				DeprecationMessage:  "Configure grantees_filter_group instead. This attribute will be removed in the next major version of the provider",
+				Validators: []validator.Set{
+					setvalidator.SizeAtLeast(1),
+				},
+			},
+			"grantees_filter_group": schema.SingleNestedAttribute{
+				MarkdownDescription: "placeholder", // TODO: Add description
+				Optional:            true,
```

Comment:
optioanl until next version?
lets put a comment why optional

Reply:
yep, its has to be optional because both fields can be used and you dont want to hard migrate the customers to the new field.

Reply:
added comment